app/static/css/custom/footer.css

```diff
@@ -53,6 +52,10 @@ footer .copyright{
 
 #footer-menu select {background: #282C35; border-color:#AAAAAA; color: #AAAAAA  }
 
+#footer-menu-options{
```

Comment:
Use `#footer-menu > div > div` ... This is more direct. `#footer-menu .row .col-sm-12` would refer to any row inside footer-menu